README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
I didn't want to list all extensions in the README. Isn't source code specific enough?

The current method used to list files included in PDB is not working with native PDBs. I'll investigate more to have a method which works in both cases.